tiledb/bioimg/converters/ome_tiff.py

```diff
@@ -21,6 +22,8 @@ def __init__(
         self,
         input_path: str,
         logger: Optional[logging.Logger] = None,
+        config: Optional[Config] = None,
+        ctx: Optional[Ctx] = None,
         extra_tags: Sequence[Union[str, int]] = (),
     ):
```

Comment:
```suggestion
        self,
        input_path: str,
        *,
        logger: Optional[logging.Logger] = None,
        config: Optional[Config] = None,
        ctx: Optional[Ctx] = None,
        extra_tags: Sequence[Union[str, int]] = (),
    ):
```
It might be worth making optional arguments keyword only so that you don't need to worry about breaking the API when adding in new optional arguments. As implemented, if someone naively tried used the old API with only positional arguments, it would just pass `extra_tags` to `config` without a clear error message.